### PR TITLE
Update enet server test.cpp

### PR DESCRIPTION
**Pull request by ipr0gr4mipas2**
Fixed most typos in the strings.

### CODE REVIEW

File: enet server test/enet server test.cpp [hunk 1]
```diff
@@ -1674,7 +1674,7 @@ void SendPacketRaw(int a1, void *packetData, size_t packetDataSize, void *a4, EN
 				name = ((PlayerInfo*)(currentPeer->data))->displayName;
 
 		}
-		GamePacket p = packetEnd(appendString(appendString(createPacket(), "OnConsoleMessage"), "`o<" + name + "`o> " + message));
+		GamePacket p = packetEnd(appendString(appendString(createPacket(), "OnConsoleMessage"), "`o<`w" + name + "`o> " + message));
```
Comment (GrowtopiaNoobs) [inline enet server test/enet server test.cpp]: I am not sure if color codes are used right

File: enet server test/enet server test.cpp [hunk 2]
```diff
@@ -3038,7 +3038,7 @@ int _tmain(int argc, _TCHAR* argv[])
 							}
 							else if (e == 3) {
 								((PlayerInfo*)(peer->data))->currentWorld = "EXIT";
-								GamePacket p = packetEnd(appendString(appendString(createPacket(), "OnConsoleMessage"), "You can't go to EXIT!"));
+								GamePacket p = packetEnd(appendString(appendString(createPacket(), "OnConsoleMessage"), "Press Back if you want to quit the world."));
```
[the remark this answers is not in the document]
Comment (ipr0gr4mipas2) [reply enet server test/enet server test.cpp]: oh yeah it isn't
made mistake, sorry :/

[the remark this answers is not in the document]
Comment (ipr0gr4mipas2) [reply enet server test/enet server test.cpp]: will make a new one

File: enet server test/enet server test.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (ipr0gr4mipas2) [reply enet server test/enet server test.cpp]: `w = white

**Pull request merged**